common/static/common/js/discussion/views/new_post_view.js

```diff
@@ -161,8 +162,17 @@
                     },
                     error: DiscussionUtil.formErrorHandler(this.$('.post-errors')),
                     success: function(response) {
-                        var thread;
+                        var thread, discussionBreadcrumbsModel;
                         thread = new Thread(response.content);
+                        // Update the breadcrumbs and discussion Ids related to current Topic
+                        if (self.discussionBoardView) {
+                            discussionBreadcrumbsModel = self.discussionBoardView.breadcrumbs.model;
+                            if (discussionBreadcrumbsModel.get('contents').length !== 0) {
```

Comment:
Just curious if `if (discussionBreadcrumbsModel.get('contents').length)`  will do what is intended here.